apps/static/dance/ai/model/cached-spacy-palette-map.json

```diff
@@ -104,7 +104,7 @@
   "output": [
     "default",
     "cool",
-    "elecronic",
+    "electronic",
```

Comment:
Is this an issue in our generating script?  We want to make sure this error doesn't repeat next time we generate the cache file.

Reply:
+1, i'm assuming we'll need to update the python script as well?

Reply:
Good catch, fixed!